source/sigv4.c

```diff
@@ -883,38 +883,6 @@ static SigV4Status_t getCredentialScope( SigV4Parameters_t * pSigV4Params,
         *canonicalURILen = index;
     }
 
-/*-----------------------------------------------------------*/
-
-    static void generateCanonicalURI( const char * pURI,
```

Comment:
Why have the `verifySigV4Parameters`, `generateCanonicalURI` and `generateCanonicalQuery` functions been removed?